Flabbergasted-Flamingos/bangazon-api
# Added non-active customer query param

# Description
Active customer query parameter

## Number of Fixes
Type of number of fixes included in this PR. 

## Related Ticket(s)
#12

## Problem to Solve
Find customers who have not placed an order

## Proposed Changes
Added functionality to list only customers who are not active

## Expected Behavior
 
Entering the url /customers/?active=false should return a list of only customers who are not active

## Steps to Test Solution

```
git fetch --all
git checkout KFComputerResource
rm -rf api/mirgations
rm db.sqlite3
python manage.py runserver
```
1. Go to the database page in your web browser
1. Enter the URL /customers/?active=false
1. Ensure that the list only returns customers whose active column is false

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
